test/parallel/test-child-process-execfile.js

```diff
@@ -2,10 +2,10 @@
 const common = require('../common');
 const assert = require('assert');
 const execFile = require('child_process').execFile;
-const path = require('path');
 const uv = process.binding('uv');
+const { fixturesPath } = require('../common/fixtures');
 
-const fixture = path.join(common.fixturesDir, 'exit.js');
+const fixture = fixturesPath('exit.js');
```

Comment:
I like option 2. It makes it obvious where the function came from without having to find the declaration and it allows us the option to continue to make `common` less monolithic.

That said, I'm actually OK with anything. Just, I like option 2 the best.